linux/usage/systemd.md

```diff
@@ -0,0 +1,59 @@
+# systemd
+
+In order to have a command or program run when the Pi boots, you can add it as a service.
+Once this is done you can start/stop enable/disable from the linux prompt
+
+## Creating a service
+
+On your Pi, create a .service file for your service e.g.
+
+myscript.service
+
+```[Unit]
+Description=My service
+After=network.target
+
+[Service]
+ExecStart=/usr/bin/python3 -u main.py
```

Comment:
One thing, you probably need to explain that service you are starting is, in this example, a Python program, but that you could have any sort of application start up here.